conference/talk_voting.py

```diff
@@ -50,6 +55,14 @@ def talk_voting(request):
                 to_attr="votes",
             )
         )
+        .annotate(
+            can_vote=Case(
+                When(created_by=request.user, then=Value(False)),
+                When(speakers__user__in=[request.user], then=Value(False)),
+                default=Value(True),
+                output_field=BooleanField(),
```

Comment:
this... looks like advanced django. :)

Reply:
More advanced than I would have liked :D